packages/server/test/integration/cypress_spec.js

```diff
@@ -99,6 +99,53 @@ const snapshotConsoleLogs = function (name) {
   return snapshot(name, stripAnsi(args))
 }
 
+function mockEE () {
```

Comment:
Just moved duplicated code to this function. The implementation was not changed.